File: src/main/java/com/jewelexx/tablocation/github/Commit.java
package com.jewelexx.tablocation.github;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Commit {
    private String sha;
    private String url;

    @JsonProperty("sha")
    public String getSHA() {
        return sha;
    }

    @JsonProperty("sha")
    public void setSHA(String value) {
        this.sha = value;
    }

    @JsonProperty("url")
    public String getURL() {
        return url;
    }

    @JsonProperty("url")
    public void setURL(String value) {
        this.url = value;
    }
}

File: src/main/kotlin/dev/cordor/tablocation/github/Commit.kt
package dev.cordor.tablocation.github

import com.fasterxml.jackson.annotation.JsonProperty

class Commit {
    @get:JsonProperty("sha")
    @set:JsonProperty("sha")
    var sHA: String? = null

    @get:JsonProperty("url")
    @set:JsonProperty("url")
    var uRL: String? = null
}
